sunkit_instruments/suvi/data/SUVI_FM2_gain.txt

```diff
@@ -0,0 +1,68 @@
+; Gain Curve for GOES-17 SUVI FM2
```

Comment:
So this is a general question for all of these text files, will they ever get updated or is it expected these files to be fixed for the rest of mission?

Reply:
That is a tough question. What is sure is that there will be additional files once GOES-T goes up. There is a chance that some of the files might change, at least for GOES-17, because the responses do not quite line up with GOES-16. So, yes, the content of that folder might change. What are the implications of that?